markdown/en/documentation/configuration.md

```diff
@@ -18,39 +38,38 @@ This chapter describes the minimum configuration necessary to search and retriev
 
 [Read Why](documentation/ready-events)
 
-We recommend placing the `ff-communication` element immediately after the `body` tag to reduce the risk of accidental reordering.
-
-It is advisable to add a comment explaining this requirement. For example:
+We recommend placing the `ff-communication` element immediately after the `body` tag to reduce the risk of accidental reordering. Furthermore we advise to add a comment explaining this requirement. For example:
 ```html
 <!-- The ff-communication element sets up the FF Web Components and must not be moved!
      All other FF Web Components must be placed hereafter. -->
 ```
 
-### Attributes
-
----
-#### **url**
-Specify the FACT-Finder endpoint. Please note the context name `/FACT-Finder-7.2`. It is not sufficient to provide the top-level domain like `http://web-components.fact-finder.de/` which will result in an error.
-
-**Note:** Make sure you are using the same protocol (http://, https://) for the `url` attribute and webapp. 
-
-Using `<ff-communication url="http://web-components.fact-finder.de/FACT-Finder-7.2" ...>` on a page hosted via **HTTPS** will result in an error.
-
----
+#### **search-immediate**
+The `search-immediate` attribute ensures, that a search is automatically triggered as soon as `ff-communication` was successfully attached to the dom. The triggered search request uses the current URL parameters. If no parameters are present, it uses `*` as query (search for all products) in conjunction with the other settings. 
```

Comment:
dom -> DOM
The triggered search request uses `ff-communication` parameters. If a parameter isn't set, default settings are used. The default search term is `*` (search for all products).